examples/blockstorage/snapshot/main-datasource.tf

```diff
@@ -0,0 +1,5 @@
+data "vkcs_blockstorage_snapshot" "recent_snapshot" {
+  name        = "snapshot-tf-example"
+  most_recent = true
+  depends_on  = [vkcs_blockstorage_snapshot.recent_snapshot]
```

Comment:
add the comment here about this dependency

Reply:
done, and in volume